docs/docusaurus/docs/core/set_up_a_gx_environment/_install_additional_dependencies/_google_cloud_platform.md

```diff
@@ -0,0 +1,37 @@
+import GxData from '../../_core_components/_data.jsx';
+import PrereqPythonInstalled from '../../_core_components/prerequisites/_python_installation.md';
+import RecommendedVirtualEnvironment from '../../_core_components/prerequisites/_recommended_virtual_environment.md';
+import InfoUsingAVirtualEnvironment from '../../_core_components/admonitions/_if_you_are_using_a_virtual_environment.md';
+
+To validate Google Cloud Platform (GCP) data with {GxData.product_name}, you create your GX Python environment, install
+{GxData.product_name} locally, and then configure the necessary dependencies.
+
+## Prerequisites
+
+- A [GCP service account](https://cloud.google.com/iam/docs/service-account-overview) with permissions to access GCP resources and storage Objects.
+- The `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. See the Google documentation [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). 
```

Comment:
```suggestion
- The `GOOGLE_APPLICATION_CREDENTIALS` environment variable is set. See [Set up Application Default Credentials](https://cloud.google.com/docs/authentication/provide-credentials-adc). 
```
YOu don't need to define the owner of the linked documentation. It is obvious when a user selects the link.

Reply:
It is, but that's after they've left our site.  It's generally considered good form to inform people in advance when clicking on a link will take them off of your site.